x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality/table/columns.tsx

```diff
@@ -85,38 +90,65 @@ const inactiveDatasetActivityColumnTooltip = i18n.translate(
   }
 );
 
-const degradedDocsDescription = (minimimPercentage: number) =>
+const degradedDocsDescription = (
+  quality: string,
+  minimimPercentage: number,
+  comparator: string = ''
+) =>
   i18n.translate('xpack.datasetQuality.degradedDocsQualityDescription', {
-    defaultMessage: 'greater than {minimimPercentage}%',
-    values: { minimimPercentage },
+    defaultMessage: '{quality} -{comparator} {minimimPercentage}%',
+    values: { quality, minimimPercentage, comparator },
   });
 
 const degradedDocsColumnTooltip = (
   <FormattedMessage
     id="xpack.datasetQuality.degradedDocsColumnTooltip"
-    defaultMessage="The percentage of degraded documents —documents with the {ignoredProperty} property— in your dataset. {visualQueue}"
+    defaultMessage="The percentage of documents with the {ignoredProperty} property in your dataset."
```

Comment:
@mdbirnstiehl do you agree with this change in the copy?

Reply:
LGTM! thanks.